doc/conf.py

```diff
@@ -105,6 +105,9 @@
 
 pygments_style = "sphinx"
 
+copybutton_prompt_text = r"\$ |uart:~\$ "
+copybutton_prompt_is_regexp = True
```

Comment:
to be consistent: add a new section for copybutton related settings (see the end of the file)

Reply:
> to be consistent: add a new section for copybutton related settings (see the end of the file)

@kartben will you make this change?